examples/07-ssh-keys.yaml

```diff
@@ -0,0 +1,34 @@
+# A simple example of ClusterConfig object:
+---
+apiVersion: eksctl.io/v1alpha4
+kind: ClusterConfig
+
+metadata:
+  name: test-cluster
```

Comment:
By current convention this would be `cluster-7` :)

Reply:
right!